10/README.md

```diff
@@ -1,20 +1,21 @@
 ```
-shortname: 10/CBD
-Name: Consolidating BigchainDB 2.0 Design
+shortname: 10/SAAR
+Name: A Strangler Application Approach to Rewriting Some Code in Go
 type: Informational
 status: Raw
 Editors: Alberto Granzotto <alberto@bigchaindb.com>
 Contributors: Vanshdeep Singh <vanshdeep@bigchaindb.com>
 ```
 
-# Consolidating BigchainDB 2.0 Design
+# A Strangler Application Approach to Rewriting Some Code in Go
 
 ## Abstract
 This document describes an experimental architecture for BigchainDB Server, illustrated in the diagram below. It replaces a small amount of Python code with Go code, allowing the BigchainDB team to experiment with Golang with low risk while achieving several other goals, including easier migration to new versions of Tendermint and higher performance.
 
 ## Motivation
 BigchainDB 2.0 Alpha 1 has been developed using BigchainDB 1.3 as a starting point. Design decisions that were correct for the first generation of BigchainDB, are now challenged by the integration of a new consensus algorithm (managed by Tendermint). Specifically, the Tendermint state machine runs in a separate process, and communicates with BigchainDB using multiple channels.
 
+## Overview
```

Comment:
I would move this subsection heading (**Overview**) to below the following paragraph, since the following paragraph lists a bunch of motivating factors. That is, move it just before the "In this BEP, we..." paragraph.